src/main/java/com/devonfw/quarkus/ordermanagement/domain/repo/ItemRepository.java

```diff
@@ -0,0 +1,7 @@
+package com.devonfw.quarkus.ordermanagement.domain.repo;
+
+import com.devonfw.quarkus.ordermanagement.domain.model.ItemEntity;
+import org.springframework.data.repository.CrudRepository;
+
+public interface ItemRepository extends CrudRepository<ItemEntity, Long>, ItemFragment {
```

Comment:
We should use JpaRepository instead of CrudRepository I think, because we also use this in the [documentation](https://github.com/GuentherJulian/devon4j/blob/jpa_guide/documentation/guide-repository.asciidoc#custom-method-implementation). Also something we have to change then in the other repo